lib/util/git/index.ts

```diff
@@ -363,18 +364,16 @@ export async function checkoutBranch(branchName: string): Promise<CommitSha> {
 
 export async function getFileList(): Promise<string[]> {
   await syncGit();
-  const branch = config.currentBranch;
-  const submodules = await getSubmodules();
-  const files: string = await git.raw(['ls-tree', '-r', branch]);
-  // istanbul ignore if
-  if (!files) {
-    return [];
-  }
+  const [submodules, files]: [string[], Dirent[]] = await Promise.all([
+    getSubmodules(),
```

Comment:
Also we can parse `.gitmodules` directly, though not sure about edge-cases of doing so

Reply:
Hm, maybe we can get rid of it, since submodules are meant to be directories and we filter them out